docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -586,6 +586,21 @@ the regular expression `/^dev-[a-z0-9-]+$/`, run the following command:
 $ tsh request create --resources /teleport.example.com/namespace/mycluster/^dev-[a-z0-9-]+$
 ```
 
+Using cluster-scoped resource requests requires the role under `search_as_roles` to have the correct 
+permissions for the cluter-scoped resource.  This is an example of the required permissions to
+request access to the namespace `prod` only:
```

Comment:
```suggestion
request access only to the namespace `prod` (and all resources within it):
```
Or maybe somehow phrase it better, but I think it would be good to explicitly mention here that access to anamespace gives access to everything in that namespace.